src/mergeSort.js

```diff
@@ -0,0 +1,27 @@
+export default function mergeSort(array) {
+
+  if(!Array.isArray(array)) return "We can only sort arrays"
```

Comment:
I strongly recommend avoiding inline control flow, preferring instead explicit braces to delineate blocks.  This improves code clarity and readability, and helps guard against possibly subtle dangling `if/else` bugs.